test/index.ts

```diff
@@ -98,24 +112,18 @@ function processEnvVars(): {
     shouldRunTypeChecks: boolean;
     shouldRunBenchmarks: boolean;
 } {
-    const isTestsEnvVarSet =
-        process.env.AZLE_INTEGRATION_TEST_RUN_TESTS === 'true';
-    const isTypeChecksEnvVarSet =
-        process.env.AZLE_INTEGRATION_TEST_RUN_TYPE_CHECKS === 'true';
-    const isBenchmarksEnvVarSet =
-        process.env.AZLE_INTEGRATION_TEST_RUN_BENCHMARKS === 'true';
-
-    const areNoVarsSet =
-        !isTestsEnvVarSet && !isTypeChecksEnvVarSet && !isBenchmarksEnvVarSet;
+    const runTests = process.env.AZLE_RUN_TESTS ?? 'true';
+    const runTypeChecks = process.env.AZLE_RUN_TYPE_CHECKS ?? 'true';
+    const runBenchmarks = process.env.AZLE_RECORD_BENCHMARKS ?? 'true';
 
-    const shouldRunTests = isTestsEnvVarSet || areNoVarsSet;
-    const shouldRunTypeChecks = isTypeChecksEnvVarSet || areNoVarsSet;
-    const shouldRunBenchmarks = isBenchmarksEnvVarSet || areNoVarsSet;
+    const hasOnly = [runTests, runTypeChecks].includes('only');
 
     return {
-        shouldRunTests,
-        shouldRunTypeChecks,
-        shouldRunBenchmarks
+        shouldRunTests: hasOnly ? runTests === 'only' : runTests !== 'false',
```

Comment:
This whole ending part is not very declarative. There's some mixing of concerns and stuff...I think it would be easier to understand potentially if variables were given the correct names...well like it was more...yes more like it was, this has been refactored to how it used to be and I don't think it's better

Reply:
I'm not sure I fully understand what you are driving at. I'll make it a bit more declarative, but I don't think we have mixing of concerns. The function is off to find one thing, which parts of the test suite should run. We have a bit of complication with isolating running only one part of it all, but hopefully that's cleared up a bit now

Reply:
done